src/Components/Nav/Nav.css

```diff
@@ -12,4 +12,62 @@ nav {
 
 .icon {
     height: 25px;
-}
\ No newline at end of file
+}
+
+.app-logo{
+    display: flex;
+    width: 55%;
+}
+
+.app-logo > h1 {
+    color: white;
+    margin: 0;
+    align-self: center;
+}
+
+
+.pear-logo {
+    height: 80%;
+    align-self: center;
+    -moz-transition: -moz-transform ease 10s;
+    -webkit-transition: -webkit-transform ease 10s;
+    -o-transition: -o-transform ease 10;
+    -ms-transition: -ms-transform ease 10s;
+    transition: transform ease 10s;
+}
+.pear-logo:active {
+    -moz-transform: rotate(3600deg);
+    -webkit-transform: rotate(3600deg);
+    -o-transform: rotate(3600deg);
+    -ms-transform: rotate(3600deg);
+    transform: rotate(3600deg);
+}
+
+.nav-buttons {
+    display: flex;
+    justify-content: space-between;
```

Comment:
They appear spaced too far apart. Update to space-around or space-evenly possibly.